node/src/main/java/eu/cloudnetservice/node/command/sub/ClusterCommand.java

```diff
@@ -128,24 +128,46 @@ public final class ClusterCommand {
       .toList();
   }
 
-  @Parser
+  @Parser(name = "anyHostAndPort")
   public @NonNull HostAndPort defaultHostAndPortParser(@NonNull CommandContext<?> $, @NonNull Queue<String> input) {
     var address = input.remove();
-    try {
-      // create an uri with the tpc protocol
-      var uri = URI.create("tcp://" + address);
+    var hostAndPort = NetworkUtil.parseHostAndPort(address, true);
+    if (hostAndPort == null) {
+      throw new ArgumentNotAvailableException(I18n.trans("command-any-host-and-port-invalid", address));
+    }
 
-      var host = uri.getHost();
-      // check if the host and port are valid
-      if (host == null || uri.getPort() == -1) {
-        throw new ArgumentNotAvailableException(I18n.trans("command-cluster-invalid-host"));
-      }
+    return hostAndPort;
+  }
 
-      var inetAddress = InetAddresses.forUriString(host);
-      return new HostAndPort(inetAddress.getHostAddress(), uri.getPort());
-    } catch (IllegalArgumentException exception) {
-      throw new ArgumentNotAvailableException(I18n.trans("command-cluster-invalid-host"));
+  @Parser(name = "assignableHostAndPort", suggestions = "assignableHostAndPort")
+  public @NonNull HostAndPort assignableHostAndPortParser(@NonNull CommandContext<?> $, @NonNull Queue<String> input) {
+    var address = input.remove();
+    var hostAndPort = NetworkUtil.parseHostAndPort(address, false);
+    // check if we can parse a host and port form the given input
+    if (hostAndPort == null) {
+      throw new ArgumentNotAvailableException(I18n.trans("command-any-host-invalid", address));
+    }
+    // check if we can assign the parsed host and port
+    if (NetworkUtil.assignableHostAndPort(hostAndPort) == null) {
+      throw new ArgumentNotAvailableException(I18n.trans("command-assignable-host-invalid", address));
     }
+    // success
+    return hostAndPort;
+  }
+
+  @Suggestions("assignableHostAndPort")
+  public @NonNull List<String> suggestAssignableHostAndPort(@NonNull CommandContext<?> $, @NonNull String input) {
+    return List.copyOf(NetworkUtil.availableIPAddresses());
```

Comment:
(Cannot comment on the method in NetworkUtil so doing it here): Shouldn't we move the `availableIPAddresses` into a static field which gets resolved only once? Especially if we have that many acesses to the list...